src/http/http_rpc_context.h

```diff
@@ -269,8 +269,16 @@ namespace http
     virtual void set_response_body(std::string&& body) override
     {
       response_body = std::vector<uint8_t>(body.begin(), body.end());
-      set_response_header(
-        http::headers::CONTENT_TYPE, http::headervalues::contenttype::TEXT);
+
+      // If there's no explicit content type already set, assume this is really
```

Comment:
I wonder if setting a default content type is the right thing to do. It seems to me that it's only made bugs slightly harder to detect for us, we would have caught them earlier if there was no default.

Reply:
I think that's a fine approach too. It changes the responses for any endpoints that were silently benefitting from this, but that's not too many, not too dangerous, and better they find out now? Yes I think I've convinced myself, going to remove the default entirely and see what breaks.